packages/calypso-e2e/src/lib/blocks/pricing-table-block.ts

```diff
@@ -47,21 +42,21 @@ export class PricingTableBlock {
 	/**
 	 * Given an appropriate value, select the gutter value on the settings sidebar for Pricing Table block.
 	 *
-	 * @param {string} value Value to set the gutter to.
+	 * @param {string} name Name of the gutter preset.
+	 * @param {number} value Value of the gutter for the "Custom" preset (Atomic).
 	 * @returns {Promise<void>} No return value.
 	 */
-	async setGutter( value: gutterValues ): Promise< void > {
+	async setGutter( name: string, value?: number ): Promise< void > {
```

Comment:
Good question. I removed it because it started getting overengineered, TBH. The `PricingTable` utility will not be used anywhere except for the few smoke tests of the CoBlocks plugin (context: p4TIVU-apL-p2, p9oQ9f-1yg-p2). The important part is verifying that those gutter settings work well on our side because we customize them to the point where they're different on Simple vs. Atomic sites. That bit is covered [within the Gutter Control extension suite](https://github.com/Automattic/wp-calypso/blob/733264178943a777815eca6d9fd64a9a02044966/test/e2e/specs/blocks/blocks__coblocks__extensions__gutter-control.ts#L54-L76), so the additional layer of conditional Simple vs. Atomic gutter typing felt overkill to me as it doesn't really bring any value at this point. Does that make sense?

Reply:
For sure, that's fine with me! 😄